.gitignore

```diff
@@ -28,3 +28,4 @@ phpunit.phar
 /phpunit.xml
 # phpunit cache
 .phpunit.result.cache
+config/.merge-plan.php
```

Comment:
Why? 
In this package merge plan do not should generated. See https://github.com/yiisoft/yii-runner-roadrunner/blob/f5f50989a5d64fd0a4d800dace666b3ab3eacf1b/composer.json#L69